lib/cable_ready/channel.rb

```diff
@@ -215,25 +29,9 @@ def add_operation(key, options)
     end
 
     def stub
-      {
-        add_css_class: [],
-        dispatch_event: [],
-        inner_html: [],
-        insert_adjacent_html: [],
-        insert_adjacent_text: [],
-        morph: [],
-        outer_html: [],
-        remove: [],
-        remove_attribute: [],
-        remove_css_class: [],
-        set_attribute: [],
-        set_cookie: [],
-        set_dataset_property: [],
-        set_style: [],
-        set_property: [],
-        set_value: [],
-        text_content: []
-      }
+      available_operations.each_with_object({}) do |available_operation, hash|
```

Comment:
I'm not 100% sure, but it seems that we're reimplementing std lib functionality here. I think that the `@operations` variable could be initialized with a hash that accepts a block, instead of receiving the return of `#stub`, like:

```ruby
# in the initializer
@operations = Hash.new { |hash, operation_name| hash[operation_name] = [] }
```

Reply:
Wow, nice call. I was pretty confident that you must be missing something, but it's working great! 😁 

It's pretty crazy how refactoring something really verbose suddenly reveals ways in which things can get even more streamlined.